vehicle/external_cmd_converter/include/external_cmd_converter/node.hpp

```diff
@@ -38,7 +40,8 @@ using ExternalControlCommand = tier4_external_api_msgs::msg::ControlCommandStamp
 using Odometry = nav_msgs::msg::Odometry;
 using raw_vehicle_cmd_converter::AccelMap;
 using raw_vehicle_cmd_converter::BrakeMap;
-using Odometry = nav_msgs::msg::Odometry;
+using ControlCommandStamped = autoware_auto_control_msgs::msg::AckermannControlCommand;
```

Comment:
Do we need this?

Reply:
It seems this code is not needed.
I will delete this line!